qa/rpc-tests/feature_issuance.py

```diff
@@ -160,5 +236,88 @@ def run_test(self):
         assert_equal(self.nodes[0].getwalletinfo()["balance"][issued["token"]], 1)
 
 
+        print("Raw issuance tests")
+        # Addresses to send to to check proper blinding
+        blind_addr = self.nodes[0].getnewaddress()
+        nonblind_addr = self.nodes[0].validateaddress(blind_addr)['unconfidential']
+
+        # Fail making non-witness issuance sourcing a single unblinded output.
+        # See: https://github.com/ElementsProject/elements/issues/473
+        total_amount = self.nodes[0].getbalance()['bitcoin']
+        self.nodes[0].sendtoaddress(nonblind_addr, total_amount, "", "", True)
+        self.nodes[1].generate(1)
+        raw_tx = self.nodes[0].createrawtransaction([], {nonblind_addr:self.nodes[0].getbalance()['bitcoin']-1})
+        funded_tx = self.nodes[0].fundrawtransaction(raw_tx)['hex']
+        issued_tx = self.nodes[2].rawissueasset(funded_tx, [{"asset_amount":1, "asset_address":nonblind_addr}])[0]["hex"]
+        blind_tx = self.nodes[0].blindrawtransaction(issued_tx)
+        signed_tx = self.nodes[0].signrawtransaction(blind_tx)
+        assert_raises_jsonrpc(-26, "", self.nodes[0].sendrawtransaction, signed_tx['hex'])
+
+        # Make single blinded output to ensure we work around above issue
+        total_amount = self.nodes[0].getbalance()['bitcoin']
+        self.nodes[0].sendtoaddress(blind_addr, total_amount, "", "", True)
+        self.nodes[1].generate(1)
+
+        # Next we used convinience function for transaction flows that will work
+
+        # Start with single issuance input, unblinded (makes 5 outputs for later larger issuances)
+        # NOTE: the "blind" argument must should for all entries for `blindrawtransaction` step
```

Comment:
This sentence doesn't seem to make sense to me.

Reply:
removed in other PR